omod/pom.xml

```diff
@@ -49,7 +49,25 @@
             <version>${webservicesRestVersion}</version>
             <scope>provided</scope>
         </dependency>
-		
+
+		<dependency>
+			<groupId>org.openmrs.module</groupId>
+			<artifactId>legacyui-api</artifactId>
+			<version>${legacyuiVersion}</version>
+			<scope>provided</scope>
+		</dependency>
```

Comment:
Any api dependency likely belongs in the api pom of the project, even if it is for a module that just provides UI components.

Reply:
Thanks @mseaton ! I have removed this dependency from the omod pom.